src/App/App.js

```diff
@@ -5,22 +5,40 @@ import About from '../About/About';
 import { Route, Switch } from 'react-router-dom';
 import Box from '@material-ui/core/Box';
 import SingleLineGridList from '../Thumbnails/Thumbnails'
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import turingSHSLogo from '../geometric-heart-logo.png'
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: '#e6ffff',
+      main: '#b3e5fc',
+      dark: '#82b3c9'
+    },
+    secondary: {
+      light: '#b6ffff',
+      main: '#81d4fa',
+      dark: '#4ba3c7'
+    },
+  },
+});
 
 function App() {
   return (
-
+  <MuiThemeProvider theme={theme}>
     <div className="App">
-      <Box component="span" m={1}>
-        <header>
-          <h1>Silver Nest</h1>
-        </header>
+      <Box component="span" display='flex' justifyContent="space-between" flexDirection="row" padding={1} m={1}>
```

Comment:
I'm really liking what Material-ui brings to the app. You can style right away, and pass down themes -- super cool!